### DISCUSSION TITLE
Grafana Missing

### DISCUSSION THREAD

**Discussion by sonicSW** [linkerd/linkerd2#7745] [Q&A] [answered]
Dear community,
I am new to linked and first of all, a big thank you! It is an amazing approach to meshed-service monitoring and management!

I have a confusion, calling linked viz dashboard, I see a Grafana should have been installed alongside the default installation steps.
I do not see the deployment/pod for this (following the default installation guide) nor can I access it. Shall I install one myself? It looks more it is deeply integrated into the viz dashboard (icon I also do not see in viz at the end of the rows).
Linkerd dashboard available at:
http://localhost:50750
Grafana dashboard available at:
http://localhost:50750/grafana

My second issue is that sometimes I lose the port-forward to the interface. Recalling the command to create and open the link exists with an error that the connection is already established (but it is not usable anymore) I help myself by calling then kubectl port-forward my self, which something works or sometimes then shows an SSL error.

Any tips to get me of the starting line are appreciated! thank you.

**Comment by alpeb** [comment, marked as answer]
Hi, are you on an edge version? We recently removed the Grafana instance that comes with Linkerd, and we're recommending instead for people to install it themselves. Here you can find some instructions on how to do that and how to link it with Linkerd (these instructions haven't been merged yet in to the published docs):
https://github.com/linkerd/website/pull/1244/files#diff-38d205e8ee0fe238c36aa26c478b23b4b4e6f52ac08c238b0767f188303ed504

**Reply by sonicSW** [reply]
Hi,

yes, my version is "Client version: edge-22.1.5". Thank you, as I was doubting my sanity.

I will just plug a Grafana into the system and see how far I get or wait for the instructions to be published.

**Reply by alpeb** [reply]
The Grafana instructions are live now at
https://linkerd.io/2/tasks/grafana/

**Comment by hummingsoul** [comment]
After connecting to an external grafana configuration,
 click the grafana icon on the interface to enter the grafana interface and prompt the dashboard not found
